docs/guides/typescript.md

```diff
@@ -445,13 +445,11 @@ const bearStore = createStore<BearState>()((set) => ({
 function useBearStore(): BearState
 function useBearStore<T>(
   selector: (state: BearState) => T,
-  equals?: (a: T, b: T) => boolean,
```

Comment:
I tried my best to update these docs to remove the `equalityFn`, but it may be worth a larger rewrite.

Reply:
This is addressed here: https://github.com/pmndrs/zustand/pull/2248